src/Microsoft.ML.Recommender/MatrixFactorizationTrainer.cs

```diff
@@ -26,11 +26,33 @@ namespace Microsoft.ML.Trainers
     /// Train a matrix factorization model. It factorizes the training matrix into the product of two low-rank matrices.
     /// </summary>
     /// <remarks>
-    /// <para>The basic idea of matrix factorization is finding two low-rank factor marcies to apporimate the training matrix.
-    /// In this module, the expected training data is a list of tuples. Every tuple consists of a column index, a row index,
+    /// <format type="text/markdown"><![CDATA[
+    /// To create this trainer, use [MatrixFactorization](xref:Microsoft.ML.RecommendationCatalog.RecommendationTrainers.MatrixFactorization(System.String,System.String,System.String,System.Int32,System.Double,System.Int32))
+    /// or [MatrixFactorization(Options)](xref:Microsoft.ML.RecommendationCatalog.RecommendationTrainers.MatrixFactorization(Microsoft.ML.Trainers.MatrixFactorizationTrainer.Options)).
+    ///
+    /// ### Input and Output Columns
+    /// There are three input columns required, one for matrix row indexes, one for matrix column indexes, and one for values (i.e., labels) in matrix .
+    /// They together defines a matrix in [COO](https://en.wikipedia.org/wiki/Sparse_matrix#Coordinate_list_(COO)) format.
+    /// The type of label column is vector of <xref:System.Single> while the other two columns are key-typed scalar.
+    ///
+    /// | Output Column Name | Column Type | Description|
+    /// | -- | -- | -- |
+    /// | `Score` | <xref:System.Single> | The predicted matrix value at the location specified by input columns (row index column and column index column). |
+    ///
+    /// ### Trainer Characteristics
+    /// |  |  |
+    /// | -- | -- |
+    /// | Machine learning task | Recommender systems |
+    /// | Is normalization required? | Yes |
+    /// | Is caching required? | Yes |
+    /// | Required NuGet in addition to Microsoft.ML | Microsoft.ML.Recommender |
+    ///
+    /// ### Background
+    /// The basic idea of matrix factorization is finding two low-rank factor marcies to apporimate the training matrix.
```

Comment:
>marcies [](start = 78, length = 7)

matrices?

Reply:
Yes!

---
In reply to: [276859081](https://github.com/dotnet/machinelearning/pull/3409#discussion_r276859081) [](ancestors = 276859081)